Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -44,8 +44,14 @@ try {
             # Delete the symbol store transaction.
             [string]$SymbolsPath = Get-VstsInput -Name 'SymbolsPath' -Require
             [string]$TransactionId = Get-VstsInput -Name 'TransactionId' -Require
-            Import-Module -Name $PSScriptRoot\PublishHelpers\PublishHelpers.psm1
-            Invoke-UnpublishSymbols -Share $SymbolsPath -TransactionId $TransactionId -MaximumWaitTime $SymbolsMaximumWaitTime -SemaphoreMessage $SemaphoreMessage
+
+            if ($SymbolsPath.StartsWith("https://")) {
+                Write-Host "Received url for FileShare deletion"
```

Comment:
shouldnt this be reverted? the server should never send the agent a deletion job for the new artifact types. deletion will be a service to service call and agent should not be involved.